Cargo.toml

```diff
@@ -41,16 +41,13 @@ bytes = "1.0.1"
 bitvec = "0.20.1"
 regex = "1.4.3"
 rand = "0.8.3"
-tiny-bip39 = "0.8.0"
+tiny-bip39 = { git = "https://github.com/Fiono11/tiny-bip39" }
```

Comment:
Creating these forks are because you can't wrap anyhow inside of a thiserror? I remember having this issue in another project.

Reply:
I think we should do everything we can to avoid making new crates because it adds to maintenance problems. Do you have an thiserror/anyhow issue that discusses this?

Reply:
Yes, I don't know if there is other way to do it. Someone opened this issue https://github.com/maciejhirsz/tiny-bip39/issues/27 but still answered. I could submit a PR there but the crate seems abandoned...maybe you could fork it instead?

Reply:
OK let's go with your fork for now, and I'll soon set up a new feeless specific crate with your fork like I did with ed25519-dalek-blake2-feeless.